domain/src/main/java/org/oppia/domain/profile/ProfileTestHelper.kt

```diff
@@ -15,15 +16,17 @@ class ProfileTestHelper @Inject constructor(
       "Sean",
       "12345",
       null,
-      allowDownloadAccess = true,
-      isAdmin = true
+      true,
+      R.color.abc_tint_default,
+      true
     )
     profileManagementController.addProfile(
       "Ben",
       "123",
       null,
-      allowDownloadAccess = false,
-      isAdmin = false
+      false,
+      R.color.abc_tint_default,
```

Comment:
Similar to earlier: here & elsewhere, suggest using the parameter names directly for clarity.

Reply:
Updated to use name arguments.